docs/make_work.jl

```diff
@@ -1,147 +1,22 @@
 using Documenter, Oscar, DocumenterMarkdown, DocumenterCitations
 
-import Documenter:
-    Anchors,
-    DocTests,
-    Documents,
-    Documenter,
-    Utilities
-
-import Documenter.Utilities: Selectors
-
-using Documenter.Builder
-import Documenter.Builder: SetupBuildDirectory, walk_navpages
-import Documenter.Documents: Document, Page
-
-#=
- copied from Documenter (0.26.1)
-
- Adds the loop to collect the AA/Nemo/Hecke doc files into the
- build tree of Oscar.
-
-=#
-function Selectors.runner(::Type{SetupBuildDirectory}, doc::Documents.Document)
-    @info "SetupBuildDirectory: setting up build directory."
-
-    # Frequently used fields.
-    build  = doc.user.build
-    source = doc.user.source
-    workdir = doc.user.workdir
-
-    # The .user.source directory must exist.
-    isdir(source) || error("source directory '$(abspath(source))' is missing.")
-
-    # We create the .user.build directory.
-    # If .user.clean is set, we first clean the existing directory.
-    doc.user.clean && isdir(build) && rm(build; recursive = true)
-    isdir(build) || mkpath(build)
-
-    # We'll walk over all the files in the .user.source directory.
-    # The directory structure is copied over to .user.build. All files, with
-    # the exception of markdown files (identified by the extension) are copied
-    # over as well, since they're assumed to be images, data files etc.
-    # Markdown files, however, get added to the document and also stored into
-    # `mdpages`, to be used later.
-    mdpages = String[]
+# Copy documentation from Hecke, Nemo, AnstratAlgebra
```

Comment:
Perhaps this should also be in `doit()`? In fact, almost everything on the top level could be moved into `doit`, so that we benefit from `Revise`.